arrays/arrays_as_function_params.c

```diff
@@ -0,0 +1,80 @@
+#include <stdio.h>
```

Comment:
Why are arrays in this PR? This PR is not related to arrays at all. There should be a separate PR for each task you do